crates/rust-analyzer/src/lsp_ext.rs

```diff
@@ -337,3 +337,31 @@ pub struct CommandLink {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub tooltip: Option<String>,
 }
+
+pub enum OpenDocs {}
```

Comment:
Let's name this `ExternalDocs` -- just `OpenDocs` should use IDE build-in facilities for viewing the docs (which VS Code lacks at the moment).